Add basic usage and enrichment notes to readme

davidstanley01 opened the pull request:
This PR adds basic usage and enrichment notes as well as links to the more thorough usage examples in the `examples` directory. 

This is mean to help users discover documentation rather than supplant the existing notebooks.

athomasoriginal commented on README.md:
````diff
@@ -1,27 +1,62 @@
+```python
+import watchful as w
+w.__version__
+#2.3.0
````
where does this come from? (the `#2.3.0`)

athomasoriginal commented on README.md:
```diff
@@ -1,27 +1,62 @@
+
+### Basic Enrichment
+
+The Watchful SDK aims to make enrichment a straight-forward task. The following step assume a brand new user without any projects.
```
`assume` -> `assumes`?

davidstanley01 replied:
should be `steps assume...`

athomasoriginal commented on README.md:
```diff
@@ -1,27 +1,62 @@
+
+The Watchful SDK aims to make enrichment a straight-forward task. The following step assume a brand new user without any projects.
+
+- Ensure you have the correct version of python, `spacey` models etc
```
What are the correct versions?

shayanjm replied:
`spacey` should be `spacy` btw

davidstanley01 replied:
Removed the word and rewrote the sentence.

athomasoriginal reviewed (approved):
I think the comments are worth fixing, but you _could_ merge without.

Also, I think we can friendly/polish these up, but my assumption is we want to get _something_ in.  If we want to nail these docs now, I can provide more thoughts.  LMK.

davidstanley01 commented:
@athomasoriginal I addressed those comments. Thank you. And, I think we do want to make a more comprehensive change to make the instructions more friendly but in a later PR when we can be a little more thoughtful. I'm definitely trying to get _**something**_ in for now.

ryttry commented:
Hey folks, thank you for pulling me into this PR. There are a few things I am adding to the PR for intentional reasons:
- Change `brew` to `python` so we don't possibly confuse the user and also to make it feel Pythonic
- Be explicit about enrichment via terminal and Jupyter Notebook, since different type of users have more affinity to one of them than the other
- Other small small changes whenever I am able to catch (I *need to be* trivial/predantic/petty as this is user-facing; I first call myself out in case you realize such detail)

ryttry commented:
@davidstanley01 thank you for beefing up the user guide
